### ISSUE TITLE
After recent changes do not know how to run project in development environment

### ISSUE THREAD

**Issue by shvetsovdm** [thoughtbot/administrate#853]
After changes in the project from November 2016 do not know how to run project on local machine.

tried:
```
$ cd spec/example_app
$ rails s
```

got:
```
/Users/Shvetsov/.rvm/gems/ruby-2.4.1@administrate/gems/rack-1.6.5/lib/rack/server.rb:354:in `initialize': No such file or directory @ rb_sysopen - /Users/Shvetsov/Projects/administrate/tmp/pids/server.pid (Errno::ENOENT)
```

tried:
```
$ foreman start
```

got:
```
worker.1 | Don't know how to build task 'jobs:work' (see --tasks)
```

using ruby-2.4.1

before starting the server I am executing `./bin/setup`

**Comment by ianagne** [comment]
My fault!  The delayed_job_active_record gem was removed as a dependency in #845 since administrate doesn't use it anywhere.  However, it looks like I forgot to remove the worker from the Procfile.

The error should go away if you delete the second line of the Procfile.  I just made a PR (#854) to fix this long term.

**Comment by shvetsovdm** [comment]
@ianagne thanks

May be it worth to add section to the readme or docs about how to start project in development, because it is not obvious for new contributor that she should use foreman for this.

**Comment by nickcharlton** [comment]
Ah, dammit! Thanks for opening this. I've merged in @ianagne's #854, so I'll close this for now.

@shvetsovdm I'd love to merge a PR that!
